lib/suma/member.rb

```diff
@@ -176,6 +181,12 @@ def replace_roles(roles)
     to_remove.each { |c| self.remove_role(c) }
   end
 
+  def create_organization_membership(organization_name)
+    organization = Suma::Organization[name: organization_name]
```

Comment:
Don't do this- just pass in the organization. This whole method is unnecessary- the caller should have an org, then it's just `m.add_organization_membership(organization: org)`.